### DISCUSSION TITLE
Wacom driver tablet settings reset on start/window focus since 2021.731.0

### DISCUSSION THREAD

**Discussion by kindlyfire** [ppy/osu#14143] [Q&A]
**Describe the bug:** I use the Linux wacom driver to set up my tablet area (using `xsetwacom`) before starting osu!. Since 2021.731.0, osu! seems to reset my area settings on start and every time I focus the window. Tablet support and high precision mouse are disabled in the settings.

This setup worked fine before 2021.731.0 (reverted to 2021.724.1 and it does not interfere with my tablet settings)

**osu!lazer version:** 2021.731.0

**Logs:** [runtime.log](https://github.com/ppy/osu/files/6944726/runtime.log)

**Comment by kindlyfire** [comment]
Someone mentioned in the original issue (I have no idea why this was moved into discussions, AFAIK this is a *bug*) that I should disable in-game tablet drivers. I re-iterate that in-game drivers *are disabled*, and that osu! did not interfere with my tablet settings until 2021.731.0.

**Reply by mk56-spn** [reply]
That was me and i deleted the message upon rereading and noticing my mistake

**Comment by bdach** [comment]
This was moved to a discussion because we cannot bear the burden of supporting every single possible Linux configuration, and so having your problem as an issue which is likely inactionable without us reproducing at least part of your setup is not productive for either us or you. Linux support takes way more resources than any other OS currently.

As to your problem, it will most likely be related to a recent SDL version bump and as such require isolating and reporting upstream. If you wish to look into it yourself then feel free to, but otherwise I wouldn't expect this to be resolved soon without any more details if I were you.
